File: org.kevoree.modeling.microframework/src/main/java/org/kevoree/modeling/api/ModelSlicer.java
package org.kevoree.modeling.api;

import org.kevoree.modeling.api.trace.TraceSequence;

import java.util.List;

/**
 * Created by duke on 9/30/14.
 */

public interface ModelSlicer {

    public void slice(List<KObject> elems, Callback<TraceSequence> callback);

}

File: org.kevoree.modeling.microframework/src/main/java/org/kevoree/modeling/api/ModelSlicer.kt
package org.kevoree.modeling.api

import org.kevoree.modeling.api.trace.TraceSequence;

/**
 * Created by duke on 9/30/14.
 */

public trait ModelSlicer {

    public fun slice(elems: List<KObject<*>>, callback: Callback<TraceSequence>);

}